nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-security/src/test/java/org/apache/nifi/web/security/headers/HTTPHeaderFiltersTest.java

```diff
@@ -99,14 +105,16 @@ public void testHSTSHeaderApplied() throws ServletException, IOException {
         FilterChain mockFilterChain = Mockito.mock(FilterChain.class);
 
         // Action
+        hstsFilter.start();
```

Comment:
I'm guessing this is needed due to API changes? seems relatively innocuous.

Reply:
yep is necessary due to some change in that class in jetty